data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,143 @@
+    "writing_system": "Syriac",
+    "script": "Estrangela",
+    "ink_color": "black",
+    "hand_note": ["Taw, Alaph, and Dalath/Resh all exhibit features of Serto."],
+    "textual_artifacts": ["ark:/21198/z1x07bdf"],
+    "extent": ["f. 1-152"]
+  }],
+  "page_layout": [{
+    "columns": 1,
+    "number_of_lines": 26,
+    "column_width": "",
+    "margin_width": "",
+    "features": [],
+    "note": [""],
+    "extent": ["f. 1-152"]
+  }],
+  "collation": {
+    "viscodex": "",
+    "formula": ["Pastedown+flyleaf, Quire 1: 1x10 (10), Quire 2: 1x6 (16), Quire 3: 1x8 (24), Quire 4: 1+1x8 (33), Quires 5-12: 8x8 (97), Quire 13: 1x8+1 (106), Quire 14: 1x6+1+1 (114), Quire 15: 1x8 (122), Quire 16: 6x1 (128), Quire 17: 1x6+1 (135), Quire 18: 1x6+1 (142), Quire 19: 1x6 (148), Quire 20 (paper): 1x4 (152)."],
+    "note": ["Quire signatures at beginning and end of most quires."],
+    "features": ["Signatures"]
+  },
+  "paratexts": [{
+      "type": "reader-note",
+      "locus": "f. 114v",
+      "transcription": {
+        "syr": ["ܢܗܘܐ ܕܘܟܪܢܐ ܛܒܐ ܩܕܡ ܐܠܗܐ ܘܡܫܝܚܗ ܠܝܥܩܘܒ ܘܠܒܪ ܫܒܘܢ ܒܢ̈ܘܗܝ ܕܒܣܝܠ ܕܡܢ ܐܝܘ ܩܪܝܬܐ"]
+      },
+      "translation": {
+        "en": ["May there be a blessed rememberance before God and his Christ for Jacob and Bar-Shabbon, sons of Basil, from the village of Yw."]
+      },
+      "note": ["The association between these persons and the scribe, or the production of the manuscript more generally, remains unclear."],
+      "features": []
+    }
+  ],
+  "associated_dates": [
+    {
+      "type": "production",
+      "date_human_readable": "6th c. CE",
+      "date_normalized": "0500/0600",
+      "source": "Kamil 1970, based on paleographic data."
+    }
+  ],
+  "associated_persons": [{
+      "type": "unknown",
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/1",
+      "attested_name_NS": "ܝܥܩܘܒ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Ya'qub bar d-Basil",
+      "source": "Attested in the note on f. 114v"
+    },
+    {
+      "type": "unknown",
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/2",
+      "attested_name_NS": "ܒܪ ܫܒܘܢ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Bar-Shabon bar d-Basil",
+      "source": "Attested in the note on f. 114v"
+    }
+  ],
+  "associated_places": [{
+      "type": "production",
+      "id": "https://sinaimanuscripts/library.ucla.edu/place/1",
+      "attested_name_NS": "ܛܘܪ ܣܝܢܝ",
+      "attested_name": "Mount Sinai",
+      "source": "Attested in the colophon on f. 152v"
+    },
+    {
+      "type": "unknown",
+      "id": "https://sinaimanuscripts/library.ucla.edu/place/2",
+      "attested_name_NS": "ܐܝܘ ܩܪܝܬܐ",
+      "attested_name": "The Village Yw",
+      "source": "Provided in connection with Jacob and Bar-Shabbon in the note on f. 114v"
+    }
+  ],
+  "provenance": [],
+  "decorations": [{
+    "note": ["Geometric decorations surround the rubrics (e.g., f. 114r)."],
+    "features": ["Decoration, Geometric"],
+    "extent": ["f. 114r"]
```

Comment:
what do we put for the extent if it’s ‘throughout’ but not necessarily on the each page? I’m not sure we want to lock ourselves into having to describe each and every instance? We could allow ‘throughout’ as a separate field (maybe a boolean), in which case we’d have the extent be the range within which the feature is ‘throughout’?

So we’d have:
```
...
"throughout": true,
"extent": "f. 1-114"
...
```

or, if it not throughout, we could have:

```
...
"throughout": false,
"extent": "f. 114v"
...
```

Reply:
Though we'll use 'locus' (or maybe 'range') instead of 'extent', the core of this question remains: how to deal with 'throughout'.